backend/app/services/llm/jobs.py

```
@@ -134,14 +137,44 @@ def execute_job(
     logger.info(
-        f"[execute_job] Starting LLM job execution | job_id={job_id}, task_id={task_id}, "
+        f"[execute_job] Starting LLM job execution | job_id={job_id}, task_id={task_id}, input_guardrail={input_guardrails}, input_query={input_query}."
     )
 
     try:
+        if input_guardrails:
+            safe_input = call_guardrails(input_query, input_guardrails, job_id)
+            logger.info(
+                f"[execute_job] Input guardrail validation | Original query={input_query}, safe_input={safe_input}/"
+            )
+            if safe_input["success"] == True and safe_input["data"]["rephrase_needed"] == False:
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `safe_input["success"]:` for truth checks

Replace with `safe_input["success"]`

(E712)